SourceX/display.cpp

```diff
@@ -178,6 +196,9 @@ bool SpawnWindow(const char *lpWindowName)
 		DvlStringSetting("scaling quality", scaleQuality, 2);
 		SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, scaleQuality);
 	} else if (fullscreen) {
+#ifdef __vita__
+		SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, 0);

```

Comment:
Let's add a DEFAULT_SCALE_QUALITY define instead.

Also, does this actually affect anything?
From what I can tell, we're setting exactly the Vita's display resolution, so no scaling should happen

Reply:
Right, that's not needed. Removed.